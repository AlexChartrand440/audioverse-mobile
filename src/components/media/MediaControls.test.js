import React from 'react'
import MediaControls from './MediaControls'

import renderer from 'react-test-renderer'

test('renders correctly', () => {
  const tree = renderer.create(<MediaControls playPause={jest.fn()} skipToPrevious={jest.fn()} skipToNext={jest.fn()} replay={jest.fn()} forward={jest.fn()} />).toJSON()
  expect(tree).toMatchSnapshot()
})
