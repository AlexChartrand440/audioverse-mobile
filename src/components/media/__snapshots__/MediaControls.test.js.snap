// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#E0E0E080",
      "flexDirection": "row",
      "justifyContent": "space-around",
      "paddingBottom": 5,
      "width": "100%",
    }
  }
>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <Image
      source={1}
      style={
        Object {
          "height": 40,
          "margin": 10,
          "tintColor": "#000000",
          "width": 40,
        }
      }
    />
  </View>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <Image
      source={1}
      style={
        Object {
          "height": 40,
          "margin": 10,
          "tintColor": "#000000",
          "width": 40,
        }
      }
    />
  </View>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#E53935",
        "borderRadius": 25,
        "height": 50,
        "justifyContent": "center",
        "opacity": 1,
        "width": 50,
      }
    }
  >
    <Image
      source={1}
      style={
        Array [
          Object {
            "height": 28,
            "tintColor": "#FFFFFF",
            "width": 28,
          },
          Object {
            "marginLeft": 1,
          },
        ]
      }
    />
  </View>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <Image
      source={1}
      style={
        Object {
          "height": 40,
          "margin": 10,
          "tintColor": "#000000",
          "width": 40,
        }
      }
    />
  </View>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <Image
      source={1}
      style={
        Object {
          "height": 40,
          "margin": 10,
          "tintColor": "#000000",
          "width": 40,
        }
      }
    />
  </View>
</View>
`;
